crates/topos-tce-api/src/stream/mod.rs

```diff
@@ -150,14 +150,25 @@ impl Stream {
                     }
                 }
 
-                Some(_stream_packet) = self.inbound_stream.next() => {
-
+                // We currently open the stream, but no other message from the client is getting processed.
+                // We are using this open connection to communicate `delivered_certificates` to the client.
+                Some(stream_packet) = self.inbound_stream.next() => {
+                    match stream_packet {
+                        Ok((_request_id, _message)) => {
+                            debug!("Received message from stream_id: {:?}", self.stream_id);
+                        }
+                        Err(error) => {
+                            // In case the stream is getting closed from the client side for example
+                            error!("Stream closed! StreamId: {}", error.stream_id);
+                            break;
```

Comment:
Will this trigger a reconnect? Or is the other side expected to reconnect?

Do you think this should be `warn` perhaps? I'm thinking that this condition will happen every time a Certificate Producer is restarting, so it's not *really* an error condition but rather something expected?

Reply:
After a round of debugging again, the `error` we receive from a disconnecting client is `TransportError`. To test this I manually stopped the `topos-sequencer`, and therefore the `stream`. I assume a "hard reboot" will lead to this error and not a nice "StreamClosed". 

There are multiple StreamErrors combined in this `error!`, the most likely a hard reboot in our case. So for now, I would leave it at this and investigate in another issue at some point how to we can get a reliable `Error` out of a disconnecting stream.